Author: Jordi Mallach <jordi@debian.org>
Description: Revert 160006402248075b95c98e9e34d4538670ba7480
 Revert upstream patch that adds -avoid-version to all libevolution
 libs, making them versionless. This is a massive WTF.
Bug-GNOME: https://bugzilla.gnome.org/show_bug.cgi?id=591436
Forwarded: no

---
--- evolution-3.13.5.orig/addressbook/gui/contact-editor/Makefile.am
+++ evolution-3.13.5/addressbook/gui/contact-editor/Makefile.am
@@ -26,7 +26,7 @@
 	e-contact-editor-dyntable.h \
 	e-contact-editor-dyntable.c
 
-libecontacteditor_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libecontacteditor_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libecontacteditor_la_LIBADD =						\
 	$(top_builddir)/e-util/libevolution-util.la			\
--- evolution-3.13.5.orig/addressbook/gui/contact-list-editor/Makefile.am
+++ evolution-3.13.5/addressbook/gui/contact-list-editor/Makefile.am
@@ -20,7 +20,7 @@
 	e-contact-list-model.c			\
 	e-contact-list-model.h
 
-libecontactlisteditor_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libecontactlisteditor_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libecontactlisteditor_la_LIBADD =				\
 	$(top_builddir)/addressbook/util/libeabutil.la		\
--- evolution-3.13.5.orig/addressbook/importers/Makefile.am
+++ evolution-3.13.5/addressbook/importers/Makefile.am
@@ -19,7 +19,7 @@
 	evolution-csv-importer.c	        \
 	evolution-addressbook-importers.h
 
-libevolution_addressbook_importers_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_addressbook_importers_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libevolution_addressbook_importers_la_LIBADD = \
 	$(top_builddir)/e-util/libevolution-util.la 			\
--- evolution-3.13.5.orig/addressbook/util/Makefile.am
+++ evolution-3.13.5/addressbook/util/Makefile.am
@@ -20,7 +20,7 @@
 	eab-book-util.c					\
 	eab-book-util.h
 
-libeabutil_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libeabutil_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libeabutil_la_LIBADD =					\
 	$(top_builddir)/e-util/libevolution-util.la	\
--- evolution-3.13.5.orig/calendar/gui/Makefile.am
+++ evolution-3.13.5/calendar/gui/Makefile.am
@@ -210,7 +210,7 @@
 	$(LIBSOUP_LIBS)							\
 	$(NULL)
 
-libevolution_calendar_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_calendar_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 EXTRA_DIST =	 			\
 	$(ui_DATA)			\
--- evolution-3.13.5.orig/calendar/importers/Makefile.am
+++ evolution-3.13.5/calendar/importers/Makefile.am
@@ -16,7 +16,7 @@
 	evolution-calendar-importer.h	     \
 	icalendar-importer.c
 
-libevolution_calendar_importers_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_calendar_importers_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libevolution_calendar_importers_la_LIBADD =			\
 	$(top_builddir)/e-util/libevolution-util.la		\
--- evolution-3.13.5.orig/composer/Makefile.am
+++ evolution-3.13.5/composer/Makefile.am
@@ -51,7 +51,7 @@
 	e-composer-text-header.c		\
 	e-msg-composer.c
 
-libevolution_mail_composer_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_mail_composer_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libevolution_mail_composer_la_LIBADD =			\
 	$(top_builddir)/e-util/libevolution-util.la	\
--- evolution-3.13.5.orig/e-util/Makefile.am
+++ evolution-3.13.5/e-util/Makefile.am
@@ -639,7 +639,7 @@
 	$(PLATFORM_SOURCES) \
 	$(NULL)
 
-libevolution_util_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_util_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libevolution_util_la_LIBADD =  \
 	$(top_builddir)/libgnomecanvas/libgnomecanvas.la \
--- evolution-3.13.5.orig/em-format/Makefile.am
+++ evolution-3.13.5/em-format/Makefile.am
@@ -98,7 +98,7 @@
 	e-mail-parser-image.c				\
 	e-mail-parser-inlinepgp-encrypted.c		\
 	e-mail-parser-inlinepgp-signed.c		\
- 	e-mail-parser-message.c				\
+	e-mail-parser-message.c				\
 	e-mail-parser-message-deliverystatus.c		\
 	e-mail-parser-message-external.c		\
 	e-mail-parser-message-rfc822.c			\
@@ -125,7 +125,7 @@
 	e-mail-stripsig-filter.c			\
 	$(SMIME_EXTENSIONS)
 
-libevolution_mail_formatter_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_mail_formatter_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 if ENABLE_SMIME
 SMIME_LIBS =						\
--- evolution-3.13.5.orig/libemail-engine/Makefile.am
+++ evolution-3.13.5/libemail-engine/Makefile.am
@@ -82,7 +82,7 @@
 	$(GNOME_PLATFORM_LIBS) \
 	$(NULL)
 
-libemail_engine_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libemail_engine_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libemail-engine.pc
--- evolution-3.13.5.orig/libgnomecanvas/Makefile.am
+++ evolution-3.13.5/libgnomecanvas/Makefile.am
@@ -54,7 +54,7 @@
 	$(GNOME_PLATFORM_LIBS) \
 	$(MATH_LIB)
 
-libgnomecanvas_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libgnomecanvas_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 BUILT_SOURCES = $(MARSHAL_GENERATED)
 
--- evolution-3.13.5.orig/mail/Makefile.am
+++ evolution-3.13.5/mail/Makefile.am
@@ -230,7 +230,7 @@
 	$(LIBSOUP_LIBS)					\
 	$(NULL)
 
-libevolution_mail_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_mail_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 noinst_PROGRAMS = test-mail-autoconfig
 
--- evolution-3.13.5.orig/mail/importers/Makefile.am
+++ evolution-3.13.5/mail/importers/Makefile.am
@@ -19,7 +19,7 @@
 	pine-importer.c				\
 	evolution-mbox-importer.c
 
-libevolution_mail_importers_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_mail_importers_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libevolution_mail_importers_la_LIBADD =				\
 	$(top_builddir)/e-util/libevolution-util.la		\
--- evolution-3.13.5.orig/shell/Makefile.am
+++ evolution-3.13.5/shell/Makefile.am
@@ -86,7 +86,7 @@
 	evo-version.h				\
 	$(NULL)
 
-libevolution_shell_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_shell_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 libevolution_shell_la_LIBADD =				\
 	$(top_builddir)/e-util/libevolution-util.la	\
--- evolution-3.13.5.orig/smime/gui/Makefile.am
+++ evolution-3.13.5/smime/gui/Makefile.am
@@ -43,7 +43,7 @@
 	$(CERT_UI_LIBS)					\
 	$(NULL)
 
-libevolution_smime_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libevolution_smime_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 ui_DATA = smime-ui.ui
 
--- evolution-3.13.5.orig/smime/lib/Makefile.am
+++ evolution-3.13.5/smime/lib/Makefile.am
@@ -35,6 +35,6 @@
 	$(CERT_UI_LIBS)				\
 	$(NULL)
 
-libessmime_la_LDFLAGS = -avoid-version $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
+libessmime_la_LDFLAGS = $(NO_UNDEFINED) $(CODE_COVERAGE_LDFLAGS)
 
 -include $(top_srcdir)/git.mk
